android/src/main/java/com/terveystalo/react_native/matomo_sdk/RNMatomoSdkModule.java

```diff
@@ -24,6 +28,8 @@
 
   private final ReactApplicationContext reactContext;
   private Tracker tracker;
+  private DimensionQueue mOneTimeDimensionQueue;
```

Comment:
I'm not sure how safe it is to reuse these dimension queues. Looking at the source https://github.com/matomo-org/matomo-sdk-android/blob/v4.1.2/tracker/src/main/java/org/matomo/sdk/extra/DimensionQueue.java#L42 it seems as if elements are removed from the queue only in cases where they don't rewrite existing values. This could lead to strange bugs where some values custom dimension values would be present in future events even though a user would have attempted to clear the value.